backend/ctgov/api/views.py

```diff
@@ -676,3 +743,514 @@ def filter_phase(request):
             q_phase |= item
 
     return q_phase
+
+
+# Function to construct elastic search filters based on search parameters posted to REST API
+def construct_elastic_filters(request, first, last):
+    study_status = (
+        request.data.get("status") if request.data.get("status") else ""
+    )
+    nct_id = (
+        str(request.data.get("nct_id")) if request.data.get("nct_id") else ""
+    )
+    condition = (
+        str(request.data.get("condition"))
```

Comment:
I don't think this cast is necessary since it should be a string already. (Also applicable to probably most or all of the other string casts)